src/country-time.coffee

```diff
@@ -97,7 +97,7 @@ infoFromLocation = (location) ->
 
   # Search timezones for match
   for zone in moment.tz.names()
-    city = zone.replace(/.*\//,'')
+    city = zone.replace(/.*\//,'').replace(/_/g, ' ')
```

Comment:
This will not work if you supply the city name with unscores. But who would do that: @koubot time in Ho_Chi_Minh